pkg/router/template/template_helper.go

```diff
@@ -358,6 +359,34 @@ func clipHAProxyTimeoutValue(val string) string {
 	return val
 }
 
+// parseIPList parses white space separated list of IPs/CIDRs (IPv4/IPv6)
+// aims at providing the same behavior as the previous approach with regexp in the template file
+func parseIPList(list string) string {
```

Comment:
Would it make sense to return "" immediately if len(string) is 0? Worried about the logging call on line 375 being spammy since its technically the "default" case. WDYT?